vault/expiration_test.go

```diff
@@ -3086,6 +3092,9 @@ func TestExpiration_getIrrevocableLeaseCounts(t *testing.T) {
 		t.Fatalf("error getting irrevocable lease counts: %v", err)
 	}
 
+	if exp.irrevocableLeaseCount != len(backends)*expectedPerMount {
```

Comment:
do we want `pendingLock` held for this access too?

Reply:
i'm thinking not based on the same logic as the above, when we're accessing this value here the irrevocable leases have all been added to the system then the counts were fetched, so at this point nothing else should be trying to access this value other than this test.

Reply:
fixed in 2558e12